src/private/conversations.ts

```diff
@@ -0,0 +1,66 @@
+import { ensureInitialized, sendMessageRequest } from "../internal/internalAPIs";
+import { GlobalVars } from "../internal/globalVars";
+import { frameContexts } from "../internal/constants";
+import { StartConversationRequest, ShowConversationRequest } from "../public/interfaces";
+
+/**
+ * Namespace to interact with the conversational subEntities inside the tab
+ */
+export namespace conversations {
+
+  /**
+  * @private
+  * Hide from docs
+  * --------------
+  * Allows the user to start a conversation with each subentity inside a tab
+  * @param startConversationRequest Callback containing the conversation Id and if the tab pane was closed
+  */
+  export function startConversation(
```

Comment:
I am still wondering if we should just merge `startConversation` and `showConversation`. They just seem too similar in functionality with the only difference being whether a `conversationId` is provided. We could just make the `conversationId` an optional parameter on the `ShowConversationRequest` structure.

Reply:
Do you remember why we decided to go w/ two different methods here?

Reply:
We went to and fro on this. The decision was that we would keep it as separate functions because

1. overloading the method makes the callback handling messy
2. "title" not making sense in the showConversation at the moment.